dbt/adapters/trino/connections.py

```diff
@@ -177,6 +178,17 @@ def open(cls, connection):
                     "http_scheme must be set to 'https' for 'kerberos' method."
                 )
             http_scheme = "https"
+        elif credentials.method == "jwt":
+            auth = trino.auth.JWTAuthentication(credentials.jwt_token)
+            if credentials.http_scheme and credentials.http_scheme != "https":
+                raise dbt.exceptions.RuntimeException(
+                    "http_scheme must be set to 'https' for 'jwt' method."
+                )
+            if credentials.jwt_token is None:
+                raise dbt.exceptions.RuntimeException(
+                    "jwt_token must be set for 'jwt' method."
+                )
+            http_scheme = "https"
```

Comment:
perhaps consider making `http_scheme`'s default `https`? this is more secure and also simplifies things. however, this would be a breaking change for existing users, but I say it is worthwhile because it makes things more secure by default.


```python
http_scheme: Optional[str] = "https"
```

there's also two `http_schema = "https"` lines on 173 and 180.

```suggestion
                    "http_scheme must be set to 'https' for 'kerberos' method."
                )
        elif credentials.method == "jwt":
            auth = trino.auth.JWTAuthentication(credentials.jwt_token)
            if credentials.http_scheme != "https":
                raise dbt.exceptions.RuntimeException(
                    "http_scheme must be set to 'https' for 'jwt' method."
                )
            if credentials.jwt_token is None:
                raise dbt.exceptions.RuntimeException(
                    "jwt_token must be set for 'jwt' method."
                )
```

Reply:
For testing purposes, Trino is running by default locally on `localhost:8080`, enabling https requires more complex configuration on server side. Let's leave configuration as simple as possible for users just to try and examine dbt-trino adapter without worrying about extra requirements.